Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/model/ConceptEntry.java

```diff
@@ -0,0 +1,301 @@
+package edu.asu.conceptpower.app.model;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+import edu.asu.conceptpower.app.constants.LuceneFieldNames;
+import edu.asu.conceptpower.app.constants.SearchFieldNames;
+import edu.asu.conceptpower.app.reflect.LuceneField;
+import edu.asu.conceptpower.app.reflect.SearchField;
+
+/**
+ * This model helps to keep track of information about a specific concept
+ * 
+ * @author Keerthivasan Krishnamurthy
```

Comment:
this is still the same class as before, right? just a different package? Let's put at least Digital Innovation Group  as author as well if there was no author information there before.